datadog_checks_dev/requirements-dev.txt

```diff
@@ -1 +1,2 @@
 -e .[cli]
+pylint
```

Comment:
You can remove this as we already have it defined in `setup.py`

Reply:
uhm tests are failing within tox in the py2 env, will double check

Reply:
added back 🤷‍♀️